src/App.vue

```diff
@@ -242,3 +245,10 @@ body > div {
   z-index: 100;
 }
 </style>
+<style lang="scss" scoped>
+.logo-icon {
+  height: 50px;
+
+  margin-top: 10px;
+}
+</style>
```

Comment:
I didn't notice during my first review, but you can also merge the two `<style>` blocks

Reply:
Done!